final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,76 @@
+#!rss_virtual_env/bin/python
+
+"""rss-reader.py 3.4
+
+Usage:
+    rss_reader.py (-h | --help)      Show help message and exit
+    rss_reader.py <rss-url>          Print rss feeds in human-readable format
+    rss_reader.py --version          Print version info
+    rss_reader.py --json             Print result as JSON in stdout
+    rss_reader.py --verbose          Outputs verbose status messages
+    rss_reader.py --limit LIMIT      Limit news topics if this parameter provided
+    rss_reader.py --date DATE        Take a date in %%Y%%m%%d format. Print cached news, published on this date.
+                                     If source argument passed, print only news from this source
+"""
+try:
+    import argparse
+    import functions.check_func as ch_f
+    import functions.process_func as proc_f
+    import functions.print_func as print_f
+    import classes.exceptions as exc
+    from functions.caching import cache_news, get_cached_news
+except ModuleNotFoundError as E:
+    print(E)
+    exit()
+except KeyboardInterrupt:
+    print(" KeyboardInterrupt")
+    exit()
```

Comment:
You do not actually need to handle `KeyboardInterrupt` exception

Reply:
Fixed